client/dashboard/sites/add-new-site/index.tsx

```diff
@@ -0,0 +1,134 @@
+import { recordTracksEvent } from '@automattic/calypso-analytics';
+import { WordPressLogo, JetpackLogo } from '@automattic/components';
+import { formatNumber } from '@automattic/number-formatters';
+import {
+	__experimentalHStack as HStack,
+	__experimentalVStack as VStack,
+	__experimentalText as Text,
+	Button,
+} from '@wordpress/components';
+import { __, sprintf } from '@wordpress/i18n';
+import { download, reusableBlock, Icon } from '@wordpress/icons';
+import devSiteBanner from 'calypso/assets/images/a8c-for-agencies/dev-site-banner.svg';
+import { preventWidows } from 'calypso/lib/formatting';
+import Column from './column';
+import MenuItem from './menu-item';
+import './style.scss';
+
+const TRACK_SOURCE_NAME = 'sites-dashboard';
+
+const wordpressClick = () => {
+	recordTracksEvent( 'calypso_sites_dashboard_new_site_action_click_add' );
+	recordTracksEvent( 'calypso_sites_dashboard_new_site_action_click_item', {
+		action: 'wordpress',
+	} );
+};
+const jetpackClick = () => {
+	recordTracksEvent( 'calypso_sites_dashboard_new_site_action_click_jetpack' );
+	recordTracksEvent( 'calypso_sites_dashboard_new_site_action_click_item', {
+		action: 'jetpack',
+	} );
+};
+const migrateClick = () => {
+	recordTracksEvent( 'calypso_sites_dashboard_new_site_action_click_item', {
+		action: 'migrate',
+	} );
+};
+const importClick = () => {
+	recordTracksEvent( 'calypso_sites_dashboard_new_site_action_click_import' );
+	recordTracksEvent( 'calypso_sites_dashboard_new_site_action_click_item', {
+		action: 'import',
+	} );
+};
+const offerClick = () => {
+	recordTracksEvent( 'calypso_sites_dashboard_new_site_action_click_item', {
+		action: 'offer',
+	} );
+};
+
+function AddNewSite() {
+	return (
+		<HStack alignment="flex-start" style={ { padding: '16px' } } spacing={ 4 }>
+			<Column title={ __( 'Add new site' ) }>
+				<MenuItem
+					icon={ <WordPressLogo /> }
+					title={ __( 'WordPress.com' ) }
+					description={ preventWidows(
+						__( 'Build and grow your site, all in one powerful platform.' )
+					) }
+					onClick={ wordpressClick }
+					href="/start?source=sites-dashboard&ref=new-site-popover"
+				/>
+				<MenuItem
+					icon={ <JetpackLogo /> }
+					title={ __( 'Via the Jetpack plugin' ) }
+					description={ preventWidows( __( 'Install the Jetpack plugin on an existing site.' ) ) }
+					onClick={ jetpackClick }
+					href={ `/jetpack/connect?cta_from=${ TRACK_SOURCE_NAME }&cta_id=add-site` }
+				/>
+			</Column>
+			<Column title={ __( 'Migrate and import' ) }>
```

Comment:
These headings likely don't match the style guide. Can we update?

| This PR | Existing Sites dashboard |
|--------|--------|
| <img width="862" alt="Screenshot 2025-05-13 at 6 40 36 AM" src="https://github.com/user-attachments/assets/2737a7b5-b601-4ee4-87e6-9429fce3f377" /> | <img width="837" alt="Screenshot 2025-05-13 at 6 42 04 AM" src="https://github.com/user-attachments/assets/4cb95b02-3179-48b1-b978-092f7badcdfa" /> |

Reply:
(maybe default styling comes around to style this later. idk).